dockers/docker-lldp/supervisord.conf.j2

```diff
@@ -87,3 +87,13 @@ stdout_logfile=syslog
 stderr_logfile=syslog
 dependent_startup=true
 dependent_startup_wait_for=lldp-syncd:running
+
+[program:container_monitor]
+command=/usr/bin/container_monitor.py lldp
+priority=6
+autostart=false
+autorestart=false
+stdout_logfile=syslog
+stderr_logfile=syslog
+dependent_startup=true
+dependent_startup_wait_for=start:exited
```

Comment:
can you insert this into the right order? after the start entry?

Reply:
ok. I will wait for more comments and get it all in one commit.